docs/experimental-features/host-risk-score.asciidoc

```diff
@@ -24,46 +24,90 @@ The following table shows how risk levels are applied to a host, based on the no
+NOTE: To enable the host risk score feature, you must have alerts in your environment. If you have previously enabled host risk score in {stack} versions 7.16 to 8.4 and are upgrading to version 8.5 or later, 
 
-To deploy the host risk score framework in your environment:
+You can enable host risk score from the following places in the  {security-app}:
+
+* The Entity Analytics dashboard 
+* The *Host risk* tab on the Hosts page
+* The *Host risk* tab on a host's details page
+
+Or, in {kib}, you can enable host risk score from Dev Tools. 
+
+TIP: You can also enable import host risk score dashboards in {kib} to analyze host risk score data. To learn how, refer to <<import-host-risk-score-dashboard>> in this topic. 
+
+To enable host risk score from the Entity Analytics dashboard: 
+
+. In the {security-app}, go to *Dashboards* -> *Entity Analytics*. 
+. In the Host Risk Scores section, click *Enable* to install the module. 
+
+To enable host risk score from the Hosts page: 
+
+. Go to *Explore* -> *Hosts*. 
+. Select the *Host risk* tab, then click *Enable* to install the module. 
 
-. https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/host-risk-score.md#8-enable-kibana-features[Enable the `riskyHostsEnabled` feature flag].
-. In the {security-app}, go to *Dashboards* -> *Overview*, then locate the *Current host risk scores* card in the lower-right corner.
-+
 [role="screenshot"]
-image::images/host-risk-score-enable-dev-tools.png[]
-. Click *Enable via Dev Tools*. This will open the Dev Tools Console in {kib}, which will be pre-populated with the scripts required to enable the host risk score feature.
-. Click the *Play* button to run each of the scripts, in their numbered order from 1 to 11.
-+
+image::images/enable-hrs.png[Enable Host Risk Score button]
+
+To enable host risk score from a host's details page: 
+
+. Go to *Explore* -> *Hosts*. 
+. Select the *All hosts* tab, then click a host name. 
+. On the details page, scroll down to the data tables, then select the *Host risk* tab. 
+. Click *Enable* to install the module. 
+
 [role="screenshot"]
-image::images/host-risk-score-dev-tools-console.png[]
+image::images/enable-hrs-details-pg.gif[Shows how to enable host risk score from the host's details page]
 
-[[import-host-risk-score-dashboard]]
+To enable host risk score from Dev Tools in {kib}, open a browser window and enter the following URL:
+
+[source,console]
+----------------------------------
+{KibanaURL}/s/{spaceID}/app/dev_tools#/console?load_from={KibanaURL}/s/{spaceID}/internal/risk_score/prebuilt_content/dev_tool/enable_user_risk_score
+----------------------------------
+
+
+NOTE: After enabling host risk score, you may get a message that says, "No host risk score data available to display." You may need to wait about an hour for data to be generated, or you can click *Restart* to force host risk score calculations to run immediately. 
```

Comment:
This maybe the part that users think this is a bug, as even users click on restart, there's no guarantee that the data will be displayed immediately. Could we rephrase a here?
Could we add more trouble shooting steps: https://github.com/elastic/security-docs/issues/2477#issuecomment-1255191055